sources/CMakeLists.txt

```diff
@@ -70,6 +76,7 @@ target_compile_definitions(picoTracker PUBLIC
   PICO_FLASH_SIZE_BYTES=16*1024*1024
   # workaround for slow init crystal on some boards see: https://github.com/raspberrypi/pico-sdk/pull/457
   PICO_XOSC_STARTUP_DELAY_MULTIPLIER=64
+  PICO_STDIO_USB_DEFAULT_CRLF=0
```

Comment:
Does this affect debug text at all?

Reply:
It shouldnt do, but I don't need this anymore now that I offset all the sent chars to avoid sending non printing chars in the messages.